style.css

```diff
@@ -231,3 +232,88 @@ span {
 .button:disabled {
   background: #2a2b33;
 }
+
+.footer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.contact {
+  color: #9390ff;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 32px;
+  justify-content: center;
+  text-align: center;
+  width: 327px;
+}
+
+.contact-text {
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  color: #f5f5ff;
+  font-family: " Inter", sans-serif;
```

Comment:
- [ ] Great job importing the "Roboto" font family with all the required weights! 💯  However, it seems that you have an extra space that is preventing you from displaying the "inter" font correctly.  Look:

- [ ] Great job importing the "Roboto" font family with all the required weights! 💯  However, it seems you forgot to import the "Inter" font family.

Look:

![image](https://user-images.githubusercontent.com/57421823/149086137-a74b3b0a-4430-4637-9d79-ce4159f49781.png)

As you can see, the browser is using Arial instead of Inter.

Kindly eliminate this extra whitespace so the font displays correctly 💪

Reply:
- [x] This well addressed and resolved ✔️